test/reducers/user.reducer.spec.js

```diff
@@ -0,0 +1,77 @@
+import userReducer, { defaultState } from '../../app/reducers/user';
+import * as userActions from '../../app/actions/user';
+import { LOADED_KNOT, RESET_STORE } from '../../app/actions/knots';
+
+const selectedTarget = {
+  name: 'target-datadotworld'
+};
+const updatedTarget = {
+  fieldValues: {
+    dataset_id: 'testing',
+    dataset_owner: '',
+    dataset_url: '',
+    api_token: ''
+  }
+};
+const targetConfig = {
+  dataset_owner: 'tester'
+};
+
+describe('user reducer', () => {
+  it('should return the initial state', () => {
+    expect(userReducer(undefined, {})).toEqual(defaultState);
+  });
+
+  it('should handle UPDATE_TARGET_FIELD', () => {
```

Comment:
Add a test spec for at least one more target to take care of a case when the target is hard coded in the reducer.